charts/hivemq-platform/values.yaml

```diff
@@ -68,6 +68,12 @@ nodes:
   #        key: my-configmap-key
   #        optional: true
 
```

Comment:
Please also change the above : Holds pod-level security attributes and common container settings. --> Configures pod-level security for the HiveMQ Platform pod.